recipes/sides/freaky-freekeh.md

```diff
@@ -0,0 +1,33 @@
+# Freaky Freekeh
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-20%25-green.svg) 
+![Speed Badge](https://img.shields.io/badge/Speed-0.3hrs-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Cheap-green.svg)
+![Serves Badge](https://img.shields.io/badge/Serves-8-green.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Green Badge Count Badge](https://img.shields.io/badge/Green%20badge%20count-100%25-green.svg)
+
++ **Chef:** Suha
++ **Sous:** Lubes
++ **Date:** Week 12 FACN2
+
+## Ingredients
+- 3 mugs of freekeh
```

Comment:
there all kinds of mugs with different shapes and sizes, so how much is a mug ?

Reply:
1 mug is more-or-less the size of 1 average-sized mug

Reply:
how much is an averaged-sized mug :rofl:

Reply:
what do you want me to change it to? cups? :D